cmd/karavictl/cmd/tenant_update.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"karavi-authorization/pb"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func NewTenantUpdateCmd() *cobra.Command {
+	tenantUpdateCmd := &cobra.Command{
+		Use:              "update",
+		TraverseChildren: true,
+		Short:            "Update a tenant resource within Karavi",
```

Comment:
please use CSM Authorization, remove references to Karavi

Reply:
done